sdk/src/test/java/co/omise/android/ConfigTest.java

```diff
@@ -0,0 +1,39 @@
+package co.omise.android;
+
+import android.os.Build;
+
+import org.junit.Test;
+
+import co.omise.android.api.Config;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConfigTest {
```

Comment:
Do you have any plan to convert it to Kotlin?

Reply:
Yeah I think with the testing task, we can rework and convert everything. Didn't want to do too much in this task.